modules/analytics/analytics.js

```diff
@@ -111,7 +111,8 @@ class Analytics {
             handlers => {
                 const permanentProperties = {
                     roomName: APP.conference.roomName,
-                    userAgent: navigator.userAgent
+                    userAgent: navigator.userAgent,
+                    ...window.jitsiRegionInfo
```

Comment:
I think I would prefer explicit declaration/copy of each attribute instead of copying whole object, which type is not defined/described anywhere